### PR TITLE
Adding a CNN-based classifier for the task of entries/other classification

### PR DESCRIPTION
**Pull request by iamgroot42**
Addresses issue https://github.com/FreeUKGen/SummerOfCodeImages/issues/2

I've  implemented a small, basic CNN for this task of classification. Images have been resized to a convenient shape, with their colors stripped off. I've added measures to take care of the small amount of data as well as the variations in it; using batch normalization, low learning rates.

In addition to this, I've also used a class-balanced error while training (to account for the class imbalance).
Using the classifier at my end, I achieved a test accuracy of 86.55% and no over-fitting.

### CODE REVIEW

File: classification/model.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Define a simple CNN model
+def getSimpleCNN(input_shape, num_classes):
+	model = Sequential()
+	model.add(Conv2D(16, kernel_size=(3, 3), input_shape=input_shape))
+	model.add(BatchNormalization())
+	model.add(Activation('relu'))	
+	model.add(Conv2D(32, (3, 3)))
+	model.add(BatchNormalization())
+        model.add(Activation('relu'))
```
Comment (benlaurie) [inline classification/model.py]: Why is indentation change not causing Python to vomit?

Comment (iamgroot42) [reply classification/model.py]: Oh well. Not sure how I missed it :(
Fixed it now
